docs/source/releasing.rst

```diff
@@ -0,0 +1,11 @@
+Making a new release
+--------------------
+
+* Make sure to populate ``CHANGELOG.md`` and change the version number
+  (see `this commit <https://github.com/beamer-bridge/beamer/commit/440b7ddffc01d16482d78ff9f18a8830670795bc>`_ for example).
+* Commit and push the changes above to Github, merge the PR and wait for the CI run to finish.
+* The previous step will have created a Docker image, which you can use for final testing before the release.
```

Comment:
It would be nice to mention the docker image and running final tests references to the agent.

Reply:
`The previous step will have created a Docker image, which you can use for final testing before the release.`
 
 What are you referring to here? I thought you are talking about an docker image for the agent which you can use for final testing. Is that right?

Reply:
I would just like to mention that it is the docker image of the agent. I'm saying that because in our mono repo lots of different modules are there (currently frontend, contracts, agent). Being a little bit more explicit would be nice I think

Reply:
> I would just like to mention that it is the docker image of the agent. I'm saying that because in our mono repo lots of different modules are there (currently frontend, contracts, agent). Being a little bit more explicit would be nice I think

Ah, I understand now, yeah, that's a good idea. :+1: